gmdn/parse.py

```diff
@@ -14,15 +16,67 @@ def startElement(self, name, attrs):
 
     def endElement(self, name):
         if name == "term":
-            print(json.dumps(self._record))
+            fp.write(json.dumps(self._record))
+            fp.write("\n")
+        else:
+            self._record[name] = ''.join(self._buffer)
+
+    def characters(self, data):
+        self._buffer.append(data)
+
+xml.sax.parse("newData/gmdnData24_6.xml", DataHandler())
```

Comment:
Should we document that you need to put the data into a folder called newData?

Reply:
I moved it, seemed a cleaner fix!